pkg/strategy/source.go

```diff
@@ -0,0 +1,50 @@
+package strategy
+
+import (
+	"strings"
+
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/types"
+	log "github.com/sirupsen/logrus"
+)
+
+type SourceFunc func(*types.KLine) fixedpoint.Value
+
+var Four fixedpoint.Value = fixedpoint.NewFromInt(4)
+var Three fixedpoint.Value = fixedpoint.NewFromInt(3)
+var Two fixedpoint.Value = fixedpoint.NewFromInt(2)
+
+type SourceSelector struct {
```

Comment:
what's this for?

Reply:
switching between different price sources from config.
for example, use `close` in config provides functionality to fetch `kline.Close` automatically and feed into indicators.

Reply:
How about moving this to the types package?